src/get_expdata.py

```diff
@@ -0,0 +1,56 @@
+'''
+Download experimental data (from hepdata and/or phenix) and write output files in jetscape-stat
+format documented [location of format specification file goes here], and/or edit its format.
+
+All relevant entries in a user-specified configuration file will be processed:
+    python get_expdata.py -c expdata_config.yaml
+'''
+    
+import os
+import yaml
+import argparse
+import get_hepdata
+import get_phenix
+import edit_data
+
+#----------------------------------------------------------------------------------------------
+def get_expdata(configFile):
+
+  # Specifiy Inputs from configFile
+  with open(configFile, 'r') as stream:
+    config = yaml.safe_load(stream)
+  
+  # Loop through configFile, and process each entry with appropriate script
+  for entry in config:
+    if 'hepdata' in entry:
+      print('Getting hepdata for entry "{}"...'.format(entry))
+      get_hepdata.get_hepdata(config[entry])
+    elif 'phenix' in entry:
+      print('Getting phenix data for entry "{}"...'.format(entry))
+      get_phenix.get_phenix(config[entry])
+    elif 'extractError' in entry:
```

Comment:
Suggest two for loops: the first one looking for hepdata and phenix, and then after all those are done, and the second for loop targeting extractError.  The advantage for this is that the extractError block may not always be in the bottom

Reply:
Actually, is it possible to control extract error from the main config block?

Something like

```
hepdata:
   ...
   ...
   extractError: [False, False, True, True]
```

Then one don't need to type out the file name "Data_ATLAS_PbPb2760_RAACharged_30to40_2015.dat" which could be less error-prone - it can be automatically determined.  What do you think?

Reply:
Hmm but then it's a bit of a mess because there are other parameters to specify.  Not sure what is the best way for this, let's just change the for-loop for now if you don't have a good alternative.

Reply:
Yea, indeed it's a bit safer to implement a second `for` loop, I'll implement that. In principle can add the extractError to the main config block as you suggest (probably with another layer of nesting), but yea I wonder if it may be a bit too fancy for the benefit...it would require a bit more structure to be imposed, and take away some of the flexibility that is there now (despite a bit of redundancy currently required).

Reply:
Yes, let’s do like this for now until we have a better alternative